src/App.tsx

```diff
@@ -1,7 +1,37 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  pressedKey: string | null;
+};
+
+export class App extends React.Component<{}, State> {
+  state = {
+    pressedKey: '',
```

Comment:
So, you put a type `string | null`
Where do you set null?)
```suggestion
    pressedKey: null,
```
Let's do it here